app/src/main/java/com/cherrish/android/presentation/challenge/mission/ChallengeMissionViewModel.kt

```
@@ -0,0 +1,65 @@
+    }
+
+    fun onAddTodoClick() {
+        val state = _uiState.value
+
+        if (state !is UiState.Success) return
+
+        if (!state.data.hasSelected) return
+
+        state.data.selectedMissions
+    }
+    fun onBackClick() {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)